rules/unique-office-backgrounds/rule.md

```diff
@@ -1,24 +1,30 @@
 ---
 type: rule
-title: Do you know your different office backgrounds?
+title: Do you know have different office backgrounds?
```

Comment:
```suggestion
title: Do you have set office backgrounds for filming?
```

Reply:
Hey @jaydenalchin 

I think we also need to remove the word "have" from you suggestion?

Reply:
In this context I was using "set" as in "have a set list of" - but it makes sense either way you'd prefer. An alternative could be "Do you _establish_ office backgrounds for filming?".